src/test/functional/features/final-order.feature

```diff
@@ -25,6 +25,23 @@ Feature: Final order
     When I click "Continue"
     Then the page URL should be "/hub-page"
 
+    Given I set the case state to "FinalOrderRequested"
```

Comment:
isn't the case state already in FinalOrderRequested as a result of the lines above?

Reply:
Removed the line that sets state